mirdata/dali.py

```diff
@@ -166,8 +173,14 @@ def audio(self):
         return load_audio(self.audio_path)
 
     def to_jams(self):
-        """(Not Implemented) Jams: the track's data in jams format"""
-        raise NotImplementedError
+        """Jams: the track's data in jams format"""
+        # Load metadata
+        metadata = {k: v for k, v in self._track_metadata.items() if v is not None}
+        y, sr = self.audio
+        metadata['duration'] = librosa.get_duration(y=y, sr=sr)
+        return jams_utils.jams_converter(
+            lyrics_data=[(self.words, None)], metadata=metadata
```

Comment:
there are multiple levels of lyrics annotations - 
```
lyrics_data = [
    (self.words, 'word-aligned lyrics'),
    (self.lines,  'line-aligned lyrics'),
    (self.paragraphs, 'paragraph-aligned lyrics')
]
```

And we should add the note data too - 
`note_data=[(self.notes, 'annotated vocal notes')]`

Reply:
thanks!